[docs] Create ios-capabilities doc for EAS Build (#13064)
expo/expo: docs/pages/build-reference/ios-capabilities.md

---
title: iOS Capabilities
---

> When you make a change to your iOS entitlements, this change needs to be updated remotely on Apple's servers before making a production build. EAS Build automatically synchronizes your changes with Apple Developer Portal when you run `eas build`!

Reviewer: ```suggestion
When you make a change to your iOS entitlements, this change needs to be updated remotely on Apple's servers before making a production build. EAS Build automatically synchronizes capabilities on the Apple Developer Portal with your local entitlements configuration when you run `eas build`.
```

> This feature can be disabled with `EXPO_NO_CAPABILITY_SYNC=1 eas build`

## Entitlements

In bare workflow apps, the entitlements are read from your `ios/**/*.entitlements` file.

In managed workflow, the entitlements are read from the introspected Expo config. You can see what your introspected config looks like by running `expo config --type introspect` in your project, then look for the `ios.entitlements` object for the results.

## Enabling

> If a supported entitlement is present in the entitlements file, then EAS Build will enable it on Apple Developer Portal. If the capability is already enabled, then EAS Build will skip it.

Reviewer: ```suggestion
If a supported entitlement is present in the entitlements file, then running `eas build` will enable it on Apple Developer Portal. If the capability is already enabled, then EAS Build will skip it.
```

Reviewer: a bit more precise to clarify that running this command is when it happens

## Disabling

> If a capability is enabled for your app remotely, but not present in the native entitlements file, then EAS Build will automatically disable it.

Reviewer: ```suggestion
If a capability is enabled for your app remotely, but not present in the native entitlements file, then running `eas build` will automatically disable it.
```
